classes/phing/tasks/ext/phpunit/PHPUnitTestRunner9.php

```diff
@@ -0,0 +1,496 @@
+<?php
+/**
+ * THIS SOFTWARE IS PROVIDED BY THE COPYRIGHT HOLDERS AND CONTRIBUTORS
+ * "AS IS" AND ANY EXPRESS OR IMPLIED WARRANTIES, INCLUDING, BUT NOT
+ * LIMITED TO, THE IMPLIED WARRANTIES OF MERCHANTABILITY AND FITNESS FOR
+ * A PARTICULAR PURPOSE ARE DISCLAIMED. IN NO EVENT SHALL THE COPYRIGHT
+ * OWNER OR CONTRIBUTORS BE LIABLE FOR ANY DIRECT, INDIRECT, INCIDENTAL,
+ * SPECIAL, EXEMPLARY, OR CONSEQUENTIAL DAMAGES (INCLUDING, BUT NOT
+ * LIMITED TO, PROCUREMENT OF SUBSTITUTE GOODS OR SERVICES; LOSS OF USE,
+ * DATA, OR PROFITS; OR BUSINESS INTERRUPTION) HOWEVER CAUSED AND ON ANY
+ * THEORY OF LIABILITY, WHETHER IN CONTRACT, STRICT LIABILITY, OR TORT
+ * (INCLUDING NEGLIGENCE OR OTHERWISE) ARISING IN ANY WAY OUT OF THE USE
+ * OF THIS SOFTWARE, EVEN IF ADVISED OF THE POSSIBILITY OF SUCH DAMAGE.
+ *
+ * This software consists of voluntary contributions made by many individuals
+ * and is licensed under the LGPL. For more information please see
+ * <http://phing.info>.
+ */
+
+/**
+ * Simple Testrunner for PHPUnit that runs all tests of a testsuite.
+ *
+ * @author  Michiel Rook <mrook@php.net>
+ * @package phing.tasks.ext.phpunit
+ */
+class PHPUnitTestRunner9 implements \PHPUnit\Runner\TestHook
+{
+    private $hasErrors = false;
+    private $hasFailures = false;
+    private $hasWarnings = false;
+    private $hasIncomplete = false;
+    private $hasSkipped = false;
+    private $hasRisky = false;
+    private $lastErrorMessage = '';
+    private $lastFailureMessage = '';
+    private $lastWarningMessage = '';
+    private $lastIncompleteMessage = '';
+    private $lastSkippedMessage = '';
+    private $lastRiskyMessage = '';
+    private $formatters = [];
+
+    /**
+     * @var \PHPUnit\Runner\TestHook[]
+     */
+    private $listeners = [];
```

Comment:
We should rename all occurences to `hook`(s) instead to cover the new behavior of phpunit.
Same for all accessors names as well.